test/test_helper.rb

```diff
@@ -45,13 +46,15 @@ def enqueue_at(job, _delay)
 class Product < ActiveRecord::Base
 end
 
-host = ENV['USING_DEV'] == "1" ? 'job-iteration.railgun' : 'localhost'
+host = ENV['USING_DEV'] == "1" ? 'job-iteration.railgun' : '127.0.0.1'
```

Comment:
job-iteration.railgun isn't going to go anywhere for us; would it be simpler to just use 127.0.0.1 for host the whole time?

Reply:
Good call. I went for the most minimal changeset but we might as well make this clean.